src/dmd/backend/cgobj.d

```diff
@@ -651,29 +677,31 @@ segidx_t OmfObj_seg_debugT()
  *      csegname        code segment name (can be null)
  */
 
-Obj *OmfObj_init(Outbuffer *objbuf, const char *filename, const char *csegname)
+Obj OmfObj_init(Outbuffer *objbuf, const(char)* filename, const(char)* csegname)
 {
         //printf("OmfObj_init()\n");
-        Obj *mobj = new Obj();
+        Obj mobj = cast(Obj)mem_calloc((void*[2]).sizeof);
```

Comment:
Wouldn't it be appropriate to use `__traits(classInstanceSize)`?

Reply:
Good idea, I'll try it out for the next one.